docs: update KeptnConfig docs to include KeptnAppCreationRequestTimeout (#1348)
keptn/lifecycle-toolkit: README.md

The `KeptnConfig` is a CRD defines configuration values for the Keptn Lifecycle Toolkit.
Currently, it can be used to configure the URL of the OpenTelemetry collector.

> Additionally it can be sed to set the interval in which automatic app discovery

Reviewer: ```suggestion
Additionally, it can be used to set the time interval in which automatic app discovery
```

> searches for workload to put into the same auto-generated KeptnApp.

Reviewer: ```suggestion
searches for workloads to put into the same auto-generated `KeptnApp`.
```

> When the parameter is not set the default value is 30 seconds.

Reviewer: ```suggestion
When the parameter is not set, the default value is 30 seconds.
```
